assets/admin/tour/style.scss

```diff
@@ -19,8 +23,7 @@ $border-color: var(--wp--preset--color--primary, currentcolor);
 	box-shadow: 0 0 $shadow-width $border-color !important;
 
 	&.sensei-tour-highlight--inset {
-		outline: none !important;
```

Comment:
Here I just took the opportunity to remove this property that is not needed anymore after https://github.com/Automattic/sensei/pull/7548. And I also removed the double `;`.